### DISCUSSION TITLE
Unable to get helix working after first time installation

### DISCUSSION THREAD

**Discussion by blitzerr** [helix-editor/helix#4208] [General] [answered]
I read about helix and wanted to try it out. I installed it using `brew` and now when I use any of these commands like
- hx --tutor

I get an error saying:

```
> hx --tutor                                                                                                                                               [2022-10-11 10:01:28 PDT] [2022-10-11 17:01:28 UTC]
Error: could not parse arguments

Caused by:
    unexpected double dash argument: --tutor
```

I thought I installed the latest version using brew but using the command `brew install helix`

```
> hx -V                                                                                                                                                    [2022-10-11 10:01:33 PDT] [2022-10-11 17:01:33 UTC]
helix-term 0.1.0
```

Is this a bug in version reporting or I installed an ancient version.

I would also love for some using helix for rust development to share their config here.

Thanks all.

**Comment by cd-a** [comment, marked as answer]
That version doesn't seem right. 

Can you do a `which hx` to see where that binary lives? Maybe you have an ancient version living in the path somewhere and the brew one wasn't linked properly.

Also check `brew info helix` to see the homebrew path. You should find it there under `/bin`

**Reply by blitzerr** [reply]
I will add some more color to this. The issue happened because a long time back I must have installed hellix using the [build from source](https://docs.helix-editor.com/install.html#build-from-source) mechanism and then I forgot about it. Today, when I installed it again using the `brew install`, it installed fine but the `**/.cargo/bin/hx` was still taking precedence. Thanks to @dahmc  for hinting towards it. So, I deleted the binary using `rm <your path>/.cargo/bin/hx` and now the latest helix, the installed using `brew` is picked up. 

```
 hx -V                                                                                                                                                    [2022-10-11 10:40:12 PDT] [2022-10-11 17:40:12 UTC]
helix 22.08.1 (66276ce6)
```

**Comment by blitzerr** [comment]
Thanks a lot @dahmc for such a quick response. You are right I am indeed using an ancient version from cargo install, which I might have done long back.

```
which hx
/Users/<>/.cargo/bin/hx
```

**Comment by blitzerr** [comment]
I removed it and now it picks up the correct version of helix. Thanks a lot @dahmc .
